app.js

```diff
@@ -1,7 +1,18 @@
+/* eslint-disable no-undef */
```

Comment:
- [ ] It is not a good practice to disable linters since they're there to help you write a better/consistent code

Reply:
Reason I used this, it is because I am calling 
"const { DateTime } = luxon;"
Which it will say luxon is undefined in the app.js file, to make sure I can use luxon without hassle I am calling the
/* eslint-disable no-undef */